test/LEGO.AsyncAPI.Tests/FluentAssersionExtensions.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) The LEGO Group. All rights reserved.
+
+namespace LEGO.AsyncAPI.Tests
+{
+    using System;
+    using System.IO;
+    using FluentAssertions;
+    using FluentAssertions.Primitives;
+    using NUnit.Framework;
+
+    /// <summary>
+    /// Contains extension methods for working with fluent assertions.
+    /// </summary>
+    internal static class FluentAssersionExtensions
+    {
+        private static readonly char[] SeperatorChars;
+
+        static FluentAssersionExtensions()
+        {
+            SeperatorChars = new[]
+            {
+                '\r',
+                '\n',
+            };
+        }
+
+        /// <summary>
+        /// Checks if the string is equal to other be ingores platform spesefic features
+        /// line new line breaks. This also checks to validate strings that are multiple lines
+        /// are the same number of lines.
+        /// </summary>
+        /// <param name="assertions">The assertion object.</param>
+        /// <param name="input">The actaul value.</param>
+        public static void BePlatformAgnosticEquivalentTo(
```

Comment:
When you have a difference it prints out this in the diff format. I don't like the built in string checks when doing multiline as it's hard to track down where the issue is. 

```diff
        The line 20 does not match
  18|              $ref: '#/components/schemas/testC'
  19|            testB:
- 20|              $ref: '#/components/schemas/testB'
+ 20|                change: I changed this
  21|components:
  22|  schemas:
  23|    testD:
```

Reply:
love it! A great addition.